### PR TITLE
use dispatch_verb()

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/busctl/busctl.c [hunk 1]
```diff
@@ -167,7 +167,9 @@ static int list_bus_names(int argc, char **argv, void *userdata) {
         if (r < 0)
                 return log_error_errno(r, "Failed to list names: %m");
 
-        pager_open(arg_no_pager, false);
+        r = pager_open(arg_no_pager, false);
```
Comment (keszybz) [inline src/busctl/busctl.c]: Hmm, do we really want this? It adds verbosity. On one hand, it's OK if the pager does not run, on the other hand, it seems correct to check if forking failed. I guess it *is* more correct to have the check.

File: src/busctl/busctl.c [hunk 2]
```diff
@@ -1978,46 +2084,26 @@ static int parse_argv(int argc, char *argv[]) {
...
-
-        if (streq(argv[optind], "help"))
-                return help();
+static int busctl_main(int argc, char *argv[]) {
+
+        static const Verb verbs[] = {
+                { "call",         4,        VERB_ANY, 0, call           },
+                { "capture",      VERB_ANY, VERB_ANY, 0, verb_capture   },
+                { "help",         VERB_ANY, VERB_ANY, 0, verb_help      },
+                { "introspect",   2,        2,        0, introspect     },
```
Comment (keszybz) [inline src/busctl/busctl.c]: `busctl introspect org.freedesktop.systemd1 /org/freedesktop/systemd1/unit/unbound_2danchor_2etimer` should work, but it doesn't after this change.

File: src/busctl/busctl.c [hunk 3]
```diff
@@ -1978,46 +2084,26 @@ static int parse_argv(int argc, char *argv[]) {
...
+                { "list",         VERB_ANY, 0,        0, list_bus_names },
```
Comment (keszybz) [inline src/busctl/busctl.c]: Hmm,
```console
$ build/busctl list
Too many arguments.
```

File: src/busctl/busctl.c [hunk 4]
```diff
@@ -1978,46 +2084,26 @@ static int parse_argv(int argc, char *argv[]) {
...
+                { "monitor",      VERB_ANY, VERB_ANY, 0, verb_monitor   },
+                { "status",       VERB_ANY, 1,        0, status         },
```
Comment (keszybz) [inline src/busctl/busctl.c]:
```console
$ build/busctl status systemd-networkd.service
Too many arguments.
```

**Comment by yuwata** [comment]
@keszybz Thank you for the comments. Updated.
- `argc` counts are fixed.
- Drop checking the returned value by `pager_open()`. Now they are voidified.

**Comment by yuwata** [comment]
Updated. Now more commands are ported to use verbs.

**Pull request closed without merging**